components/general/BottomNav.jsx

```
@@ -36,15 +36,15 @@ export default function BottomNav({ tabsState, onTabClick }) {
           <TooltipTrigger>
             <div
               onClick={() => onTabClick("technologies")}
-              className="p-3 text-md flex flex-col items-center justify-center gap-1 text-black cursor-pointer"
+              className="p-3 text-md flex flex-col items-center justify-center gap-1 text-white cursor-pointer"
             >
```

Biome (1.9.4)

[error] Enforce to have the onClick mouse event with the onKeyUp, the onKeyDown, or the onKeyPress keyboard event.

Actions triggered using mouse events should have corresponding keyboard events to account for keyboard-only navigation.

(lint/a11y/useKeyWithClickEvents)